### PR TITLE
DIG-55 upload training data

### PR DESCRIPTION
**Pull request by OFranke**

### CODE REVIEW

File: data/data_banknote_authentication.csv [hunk 1]
```diff
@@ -0,0 +1,1372 @@
+3.6216,8.6661,-2.8073,-0.44699,0
+4.5459,8.1674,-2.4586,-1.4621,0
+3.866,-2.6383,1.9242,0.10645,0
+3.4566,9.5228,-4.0112,-3.5944,0
+0.32924,-4.4552,4.5718,-0.9888,0
+4.3684,9.6718,-3.9606,-3.1625,0
+3.5912,3.0129,0.72888,0.56421,0
+2.0922,-6.81,8.4636,-0.60216,0
```
Comment (OFranke) [inline data/data_banknote_authentication.csv:8]: invalid data

**Review by svb88 — APPROVED** [review]

File: data/data_banknote_authentication.csv [hunk 2]
```diff
@@ -0,0 +1,1372 @@
+3.6216,8.6661,-2.8073,-0.44699,0
+4.5459,8.1674,-2.4586,-1.4621,0
+3.866,-2.6383,1.9242,0.10645,0
```
Comment (ChristianCKKoch) [inline data/data_banknote_authentication.csv:3]: test...

File: data/data_banknote_authentication.csv (hunk 1 above)
Comment (ingmar-moust) [reply data/data_banknote_authentication.csv:8]: test a comment

Comment (OFranke) [reply data/data_banknote_authentication.csv:8]: solved

**Review by ChristianCKKoch — APPROVED** [review]
alles gut

**Pull request merged**